geoportailv3/static/js/infobar/elevationdirective.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @fileoverview This file provides a "elevation" directive. This directive is
+ * used to insert Elevation information into the HTML page.
+ * Example:
+ *
+ * <app-elevation app-elevation-map="::mainCtrl.map"map></app-elevation>
+ *
+ * Note the use of the one-time binding operator (::) in the map expression.
+ * One-time binding is used because we know the map is not going to change
+ * during the lifetime of the application.
+ *
+ */
+goog.provide('app.elevationDirective');
+
+goog.require('app');
+goog.require('app.projections');
+goog.require('ngeo.Debounce');
+
+
+/**
+ * @return {angular.Directive} The Directive Object Definition.
+ * @ngInject
+ */
+app.elevationDirective = function() {
+  return {
+    restrict: 'E',
+    scope: {
+      'map': '=appElevationMap'
+    },
+    controller: 'AppElevationController',
+    controllerAs: 'ctrl',
+    bindToController: true,
+    template: '<span class="elevation" translate>' +
+                'Elevation: {{elevation}} m</span>'
```

Comment:
You use `controllerAs: 'ctrl'` (which is goog), so you want to use `{{ctrl.elevation}}` in the template. This requires placing `elevation` on the controller instead of directly on the scope.